ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleBannerAd.java

```diff
@@ -1,121 +0,0 @@
-package com.google.ads.mediation.vungle;
```

Comment:
Why we removed this class? Propose only remove VungleNativeAd related code parts.